migrations/capcons/migration_test.go

```diff
@@ -1436,6 +1517,10 @@ func TestLinkMigration(t *testing.T) {
 					capabilityID: 2,
 				},
 			},
+			expectedEvents: []string{
```

Comment:
Are these events get emitted during the actual migration as well?
(was thinking if that could be an overhead for the migration)

Reply:
In the tests here, I'm collecting the events for the assertions. In the real migration code in flow-go, we can just stub out `CapabilityControllerIssueHandler`'s `EmitEvent` and make it a noop.